build/python/build/autotools.py

```diff
@@ -44,6 +44,11 @@ def configure(self, toolchain, src=None, build=None):
             build = self.make_build_path(toolchain)
 
         if self.autogen:
+            # run autogen.sh in wolfssl
```

Comment:
What's that, and why do you hide it in this commit?

Reply:
wolfssh source is built by
./autogen.sh
./configure
make
as documented in its INSTALL file instead of standard automake which is expected on autotools.py.